### PR TITLE
Remove the ex which not thrown

### PR DESCRIPTION
**Pull request by wayilau**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [x] Bug fix
- [ ] New feature provided
- [ ] Improve performance

Remove the ex which not thrown. & add logs for interceptor.

### CODE REVIEW

File: apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/loader/InterceptorInstanceLoader.java [hunk 1]
```diff
@@ -80,6 +78,9 @@
             }
             if (inst != null) {
                 INSTANCE_CACHE.put(instanceKey, inst);
+                logger.debug("interceptor instance is created for class '{}', instanceKey is: {}", className, instanceKey);
+            }else {
```
Comment (wu-sheng) [inline apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/loader/InterceptorInstanceLoader.java]: `inst != null` is not a double-check, if `inst` can't be created by `inst = Class.forName(className, true, pluginLoader).newInstance();`, in most cases, exception should be thrown.

File: apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/loader/InterceptorInstanceLoader.java [hunk 2]
```diff
@@ -80,6 +78,9 @@
             }
             if (inst != null) {
                 INSTANCE_CACHE.put(instanceKey, inst);
+                logger.debug("interceptor instance is created for class '{}', instanceKey is: {}", className, instanceKey);
```
Comment (wu-sheng) [inline apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/loader/InterceptorInstanceLoader.java]: What is the purpose of adding a log for this? I think even most agent developer doesn't care much about this.

Comment (wu-sheng) [reply apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/loader/InterceptorInstanceLoader.java]: I don't know whether we need that. I am OK to discuss. Right now, there are only @ascrutae @candyleer and I have experiences and developed core codes. The opinion of mine is, most people even don't understand what the agent really does.

Comment (wayilau) [reply apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/loader/InterceptorInstanceLoader.java]: I've remove the logger just now. we can add this later in sometime when more people wants to learn more about the detail work what the agent does.

Comment (wayilau) [reply apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/loader/InterceptorInstanceLoader.java]: > I've remove the logger just now. we can add this later in sometime when more people want to learn more about the detail work what the agent does.

**Pull request merged**